app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/supportingevidence/scan_progress.scala.html

```diff
@@ -38,23 +38,20 @@
 
 @(upscanUpload : UpscanUpload)(implicit request:RequestWithSessionData[_], messages:Messages, appConfig: ViewConfig)
 
-    @key = @{"supporting-evidence.check-upscan-status"}
-    @cyaKey = @{"supporting-evidence.check-your-answers"}
+    @key = @{"supporting-evidence.scan-progress"}
     @title = @{messages(s"$key.title")}
 
     @mainTemplate(title = title, userType = request.userType) {
 
-        @caption(messages(s"$cyaKey.caption"))
+        @caption(messages(s"$key.caption"))
         @pageHeading(title)
 
-            <p>
-                @{messages(s"$key.in-progress.help-text")}
-            </p>
-
-            @formWrapper(routes.SupportingEvidenceController.uploadSupportingEvidenceCheckSubmit(upscanUpload.uploadReference.value), 'novalidate -> "novalidate") {
-             @submitButton("Check and continue")
-            }
+        <p>@{messages(s"$key.p1")}</p>
+        <p>@{messages(s"$key.p2")}</p>
 
+        @formWrapper(routes.SupportingEvidenceController.scanProgressSubmit(upscanUpload.uploadReference.value), 'novalidate -> "novalidate") {
+          @submitButton("Continue")
```

Comment:
this should come from the messages file - there should already be a key which is something like `button.continue`

Reply:
fixed